apps/federatedfilesharing/lib/Controller/RequestHandlerController.php

```diff
@@ -411,10 +371,10 @@ public function updatePermissions(int $id, ?string $token = null, ?int $permissi
 	/**
 	 * translate Nextcloud permissions to OCM Permissions
 	 *
-	 * @param $ncPermissions
+	 * @param int|null $ncPermissions
 	 * @return array
 	 */
-	protected function ncPermissions2ocmPermissions($ncPermissions) {
+	protected function ncPermissions2ocmPermissions(?int $ncPermissions = null): array {
```

Comment:
null is not supported by the code.
```suggestion
	protected function ncPermissions2ocmPermissions(int $ncPermissions): array {
```

(Also adjust the doc block)